src/redux/configureStore.js

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable */
```

Comment:
- [ ]  I will strongly advise you to remove all commented code in your codebase so as to make your codebase clean and easy to maintain by anyone that will be working on your project 👍🏽 . Kindly work on it 😄

Reply:
Thank you! All issues resolved.